src/Component/Player.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import PlayerHealth from 'Component/PlayerHealth'
+
+function Player(props) {
```

Comment:
```suggestion
function Player({ type, health }) {
```
- props destructuring
- prop name should be type and health as it is already player component

Reply:
Done